generators/generator-base-private.js

```diff
@@ -887,11 +887,17 @@ module.exports = class JHipsterBasePrivateGenerator extends Generator {
       } else {
         const javaVersion = stderr.match(/(?:java|openjdk) version "(.*)"/)[1];
         if (
+          !javaVersion.match(new RegExp('16'.replace('.', '\\.'))) &&
+          !javaVersion.match(new RegExp('15'.replace('.', '\\.'))) &&
+          !javaVersion.match(new RegExp('14'.replace('.', '\\.'))) &&
+          !javaVersion.match(new RegExp('13'.replace('.', '\\.'))) &&
           !javaVersion.match(new RegExp('12'.replace('.', '\\.'))) &&
           !javaVersion.match(new RegExp('11'.replace('.', '\\.'))) &&
           !javaVersion.match(new RegExp(constants.JAVA_VERSION.replace('.', '\\.')))
```

Comment:
Default is duplicated Java 11
```suggestion
          !javaVersion.match(new RegExp('8'.replace('.', '\\.')))
```